Sources/Crashes/CrashCollection.swift

```diff
@@ -38,9 +41,12 @@ public enum CrashCollectionPlatform {
 @available(iOS 13, macOS 12, *)
 public final class CrashCollection {
 
-    public init(platform: CrashCollectionPlatform) {
-        crashHandler = CrashHandler()
-        crashSender = CrashReportSender(platform: platform)
+    public init(crashReportSender: CrashReportSending,
+                crashCollectionStorage: KeyValueStoring = UserDefaults()) {
```

Comment:
Let's swap this out with the `standard` instance:

```suggestion
                crashCollectionStorage: KeyValueStoring = UserDefaults.standard) {
```